.circleci/config.yml

```diff
@@ -11,29 +11,46 @@ executors:
     environment:
       CGO_ENABLED: 0
 
+commands:
+  gomod:
+    steps:
+      - restore_cache:
+          keys: [v1-gomod-]
```

Comment:
@yundt We could if we wanted to be sure to grab the exact cache, or fall back to this key without a checksum -- but either way this will ensure we get the latest cache (which is probably sufficient for our use case)

https://circleci.com/docs/2.0/caching/#restoring-cache